_targets.R

```diff
@@ -4,39 +4,50 @@ source("2_process/src/process_and_style.R")
 source("3_visualize/src/plot_timeseries.R")
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
+tar_option_set(packages = c("tidyverse", "dataRetrieval", "retry")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  #map over site list downloading each site, with retry; then combine
+  #or just download in one web service call...
+  
+  tar_target(
+    sites,
+    c("01427207", "01432160", "01435000", "01436690", "01466500")
+  ),
+  tar_target(
+    site_data_frames,
+    retry(get_nwis_site_data(sites, parameterCd = '00010', 
+                             startDate="2014-05-01", endDate="2015-05-01"),
+          max_tries = 4,
+          when = 'failed',
+          interval = 0.5),
```

Comment:
^ hehehe you know too much! You are supposed to suffer with this until we get to it later 😉